config/common.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
-export DM_API_DOMAIN=${DM_API_DOMAIN:=http://localhost:5000}
+export DM_ENVIRONMENT=${DM_ENVIRONMENT:="development"}
```

Comment:
Does `DM_ENVIRONMENT` need to be in common? We're setting the default to `development` in `env.rb` anyway?

Reply:
I put it there so that the definitions following it in the same file reusing `DM_ENVIRONMENT` don't fail to blank spaces, which can be confusing and hard to debug.